unity3d/Assets/Swrve/SwrveSDK/SwrveSDK.cs

```diff
@@ -1323,20 +1323,20 @@ public string GetAppStoreLink (int gameId)
     /// <returns>
     /// In-app message for the given event.
     /// </returns>
-	public SwrveMessage GetMessageForEvent (string eventName) {
-		if (!checkCampaignRules (eventName, SwrveHelper.GetNow())) {
-			return null;
-		}
+	  public SwrveMessage GetMessageForEvent (string eventName, IDictionary<string, string> payload=null) {
```

Comment:
Excuse my lack of unity knowledge here, but are we changing the signature of a public method here? Do our customers use this method?